### DISCUSSION TITLE
Is the catcher's speed slower than in stable?

### DISCUSSION THREAD

**Discussion by Doxxz** [ppy/osu#28354] [Q&A]
Was giving lazer a go and noticed ctb feels different to play than in stable. It feels like you have to move earlier than in stable to catch the same patterns. I asked around a bit and it seems like it's not just me that feels like this.

Could it be that the catcher moves slower in general?

Or maybe hyperfruits give a weaker speed boost so you need to move earlier to compensate?

In catch, if you hold both movement keys, the catcher stops moving. Say you are moving right and want to move left next, you need to release right entirely before you can go left. So could it be that releasing in lazer is slower, making you need to release earlier?

**Comment by peppy** [comment]
There should not be any issue like this, no.

We're open to a more analytical approach to reporting this though. ie. if you or others frame-by-frame compare stable to lazer and find a discrepancy we can investigate further.

**Reply by FedSilVor** [reply]
I think it's more about the input-lag being different, that makes the catcher look slower. I also feel a very significant difference.

**Reply by peppy** [reply]
Again, it shouldn't be. We would need empirical testing by you to show this. We've done our own testing and it's on-par as far as we can tell.

**Reply by Tian-MC** [reply]
Hi peppy, I figured out the issue today.
It seems to be caused by Windows 11 (and recent versions of Windows 10) fullscreen optimizations.
This might be related to how some graphics backends or rendering engines interact with that feature.
I was able to fix it by enabling "Disable fullscreen optimizations" in the osu! executable’s compatibility settings and turning on Raw Input in the game settings.
After that, the input and performance issues were completely gone.
